### DISCUSSION TITLE
Listen for navigation

### DISCUSSION THREAD

**Discussion by HenrijsS** [sveltejs/kit#3060] [Q&A]
Is there a way to listen to page navigation?

I have a mobile menu that opens based on a `let navOpened` value.

I need to change it to false upon page navigation. Is this possible in sveltekit?
Of course, one option is to bind a click handler on all `a` tags, but that gets repetitive for each link.

**Comment by bluwy** [comment]
You can use the [`sveltekit:navigation-start` event](https://kit.svelte.dev/docs#events), or the [$navigating store](https://kit.svelte.dev/docs#modules-$app-stores) for this.
